### ISSUE TITLE
FR: Tray menu suggession.

### ISSUE THREAD

**Issue by darshinimashar** [tailscale/tailscale#3319]
### Tell us about your idea!

Tray menu suggession.

### What are you trying to do?

In the current app, with only personal devices, all devices are buried under (Network Devices -> My Devices -> ...).

It might be nicer in all cases to bring My Devices to the top-level of the menu, adjacent to Network Devices, e.g.:
- This Device (...)
- My Devices ->
- Network(?) Devices ->

### How should we solve this?

Improve the tray menu > My Devices 

### What is the impact of not solving this?

It's hard to navigate to the devices.

**Comment by jdgiddings** [comment]
This has bothered me as well, especially once networks get large the small tray menus get very difficult to navigate (this is actually what prompted me to set up ACLs)

Perhaps ditching the tray menu entirely for a more fleshed out windowed GUI?

**Comment by rosszurowski** [comment]
Yeah, the whole tray menu is kind of lacking. We have thoughts on how to improve it, just need to prioritize it against other things. Thanks for the input!

**Comment by jhult** [comment]
Somewhat related (on the tray usability front): https://github.com/tailscale/tailscale/issues/3183

**Comment by jrk** [comment]
When there are relatively few devices, they could also be flattened into a single Network Devices menu with named *sections* within that menu for what would otherwise have been submenus for "My Devices," etc. So `[Tailscale] > Network devices` would directly be a menu of devices, but with sections for different subnets (presumably with a header naming the subnet).
